pytorch_lightning/accelerators/accelerator.py

```diff
@@ -114,7 +114,7 @@ def pre_dispatch(self, trainer: 'pl.Trainer') -> None:
     def _move_optimizer_state(self) -> None:
         """ Moves the state of the optimizers to the GPU if needed. """
         for opt in self.optimizers:
-            state = defaultdict(dict)
+            state: DefaultDict = defaultdict(dict)
```

Comment:
sent this out as a different PR here: #7318
but wanted it here to ensure tests were passing